css-pseudo-classes/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>CSS PSEUDO CLASSES</title>
+</head>
+<body>
+  <h1>Hover Button</h1>
+  <button class="blackButton">I'm a button!</button>
+  <h2>Striped Table</h2>
+  <table>
+    <thead>
+      <tr>
+        <th>Student Name</th>
+        <th>Course</th>
+        <th>Grade</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr class="shade">
```

Comment:
this needs to be done using the `nth-child` selector as noted in the instructions, please update!